pkgs/build-support/dotnet/build-dotnet-global-tool/update.sh

```diff
@@ -12,9 +12,9 @@ nixeval() {
 
 nugetName=$(nixeval "$attr.nupkg.pname")
 
-# always skip prerelease versions for now
+# allow prerelease versions
 version=$(curl -fsSL "https://api.nuget.org/v3-flatcontainer/$nugetName/index.json" |
-    jq -er '.versions | last(.[] | select(match("^[0-9]+\\.[0-9]+\\.[0-9]+$")))')
+    jq -er '.versions | last(.[] | select(match("^[0-9]+\\.[0-9]+\\.[0-9]+(-[0-9A-Za-z.-]+)?(\\+[0-9A-Za-z.-]+)?$")))')
```

Comment:
I don't think we want to do this by default.  This will cause all the existing packages to update to pre-release automatically.

If we're going to do this it should be gated by something like `allowPrerelease`, but it may not be worth the trouble.  We may not want things doing automatic updates to prerelease versions at all.

Reply:
I think my suggestion would be to just leave this out for now.